scripts/transform.sh

```diff
@@ -1,11 +1,34 @@
 #!/bin/bash
 
+# To run this script in development, first build the following packages:
+#     packages/@aws-cdk/assert
+#     packages/aws-cdk-lib
+#     tools/individual-pkg-gen
+
 set -euo pipefail
 scriptdir=$(cd $(dirname $0) && pwd)
 
 # Creates a symlink in each individual package's node_modules folder pointing
 # to the root folder's node_modules/.bin. This allows yarn to find the executables
 # it needs (e.g., jsii-rosetta) for the build.
+#
+# The reason yarn doesn't find the executables in the first place is that they are
+# not dependencies of each individual package -- nor should they be. They can't be
```

Comment:
This doesn't tell the whole story (for example, Yarn _does_ find the outer monorepo private executables like `cdk-build`).

Do you know why `jsii-rosetta` is different than the packages in `tools/`?

Reply:
Most likely because the script `build` is just `cdk-build`, but the script `rosetta:extract` is `yarn --silent jsii-rosetta extract`. I tried to remove the `yarn` part, but that's apparently another can of worms. So it will go to the "ideas to try in the future" section.